packages/compass-crud/package.json

```diff
@@ -42,10 +42,9 @@
     "typecheck": "tsc --noEmit",
     "eslint": "eslint",
     "prettier": "prettier",
-    "lint": "npm run eslint .",
-    "prettier-skipped-for-now": "&& npm run prettier -- --check .",
+    "lint": "npm run eslint . && npm run prettier -- --check .",
     "depcheck": "compass-scripts check-peer-deps && depcheck",
-    "check": "npm run typecheck && npm run lint && npm run depcheck",
+    "check": "npm run lint && npm run depcheck",
```

Comment:
Why is the typecheck command removed from the check script? This PR seems to have typescript files added to the package source

Reply:
Nice catch, this was a merge conflict as we just reformatted this file and updated this check to use prettier. Re-added it 👍